.expeditor/config.yml

```diff
@@ -44,3 +44,4 @@ pipelines:
       description: Deploy the website
       definition: .expeditor/website.pipeline.yml
       trigger: pull_request
+  - habitat/build
```

Comment:
I assume this maps to `build.habitat.yml`, yeah? If so, why the inversion in the name?

Reply:
Yep - that's the default translation: https://expeditor.chef.io/docs/integrations/buildkite/#default-definition-values

Reply:
More context: `habitat` is the _type_ of pipeline and `build` is the name.